threedi_schema/tests/test_migration.py

```diff
@@ -224,6 +224,7 @@ def test_tables(self, schema_ref, schema_upgraded):
         # and whether the removed tables are not present*
         tables_new = set(get_sql_tables(get_cursor_for_schema(schema_upgraded)))
         assert self.added_tables.issubset(tables_new)
+        # breakpoint()
```

Comment:
this should no longer be necessary

Reply:
see https://github.com/nens/threedi-schema/pull/139#discussion_r1858261847